config/jobs.yaml

```diff
@@ -1601,6 +1601,15 @@ jobs:
       collections: landlock
     kcidb_test_suite: kselftest.landlock
 
+  # No hugepages allocation (for architectures without it)
+  kselftest-mm:
+    <<: *kselftest-job
+    params:
+      <<: *kselftest-params
+      extra_kernel_args: "secretmem.enable kpti=off"
```

Comment:
kernelci/kernelci-core#2828 will introduce support for passing `extra_kernel_args`